include/yarp/unescape.h

```diff
@@ -32,6 +32,6 @@ __attribute__((__visibility__("default"))) extern void
 yp_unescape_manipulate_string(const char *value, size_t length, yp_string_t *string, yp_unescape_type_t unescape_type, yp_list_t *error_list);
 
 __attribute__((__visibility__("default"))) extern int
-yp_unescape_calculate_difference(const char *value, size_t length, yp_unescape_type_t unescape_type, yp_list_t *error_list);
+yp_unescape_calculate_difference(const char *value, size_t length, yp_unescape_type_t unescape_type, bool expect_single_codepoint, yp_list_t *error_list);
```

Comment:
It's a straightforward and dirty way - just to add an argument.

Wondering if it makes sense to have an additional  `yp_unescape_type_t` constant (e.g. `YP_UNESCAPE_SINGLE`) and `yp_unescape_flag_t` constant (e.g. `YP_UNESCAPE_FLAG_SINGLE`)